modules/internal/LocaleTree.chpl

```diff
@@ -51,4 +51,51 @@ module LocaleTree {
   }
 
   chpl_initLocaleTree();
+
+  iter localeTree(root = here, parent = here) : locale {
+    halt("DONT USE SERIAL ITER");
+    yield parent;
+  }
+
+  /*
+   Logarithmic gather iterator
+   */
+  iter localeTree(param tag:iterKind, root = here, parent = here) : locale where tag == iterKind.standalone {
```

Comment:
I think it would be good to call this something other than ``localeTree``. I would want something called ``localeTree`` to actually do an on-statement onto each locale.